R/flex_dashboard.R

```diff
@@ -355,7 +349,7 @@ flex_dashboard <- function(fig_width = 6.0,
        '<script type="text/javascript">',
        '$(document).ready(function () {',
        '  FlexDashboard.init({',
-       paste0('    theme: "', theme, '",'),
+       paste0('    theme: "', if (is.character(theme)) theme else 'cosmo', '",'),
```

Comment:
Just curious - if `theme` is `"default"`, does that have the same effect as `"cosmo"`?

Reply:
Yea, that's the intention. Just FYI, I decided to ultimately get rid of this client-side accent color resolving logic (and instead do all the resolving server-side), which you'll see in the next PR.